src/App.tsx

```diff
@@ -10,7 +10,7 @@ import GovernanceTools from 'components/GovernanceTools';
 import { WalletContext } from 'lib/wallet';
 import 'styles/globals.css';
 
-const supportedNetworks = ['main', 'emerynet', 'ollinet', 'devnet', 'local'];
+const supportedNetworks = ['main', 'emerynet', 'ollinet', 'devnet', 'xnet', 'local'];
```

Comment:
should this list come from agoric-sdk?

Reply:
I think that would be cool - either that, or some resolution on main.agoric.net either on the network-config as a new key or a new endpoint